server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/measure/MeasureRepositoryImpl.java

```diff
@@ -21,8 +21,16 @@
 
 import com.google.common.base.Optional;
 import com.google.common.collect.SetMultimap;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.HashMap'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)